src/pages/DatasetRegistration.js

```diff
@@ -1143,7 +1143,7 @@ class DatasetRegistration extends Component {
                                 isDisabled: isUpdateDataset || !diseases,
                                 isMulti: true,
                                 loadOptions: (query, callback) => this.searchOntologies(query, callback),
-                                onChange: (option) => this.onOntologiesChange(option),
+                                onBlur: (option) => this.onOntologiesChange(option),
```

Comment:
This is a real-time update so this **should** be onChange, not onBlur. With each keystroke you provide a new string to filter out the preset list of ontologies obtained via API. The code also breaks if you select a disease with onBlur, which means the AsyncSelect component probably does not create an initial list, hence the reason why React errors out (Can't called map on undefined).

Reply:
Ah whoops, nice catch!